plugins/PostProcessingPlugin/scripts/LimitXYAccelJerk.py

```diff
@@ -31,15 +32,19 @@ def initialize(self) -> None:
         self._instance.setProperty("y_jerk", "value", jerk_print_old)
         ext_count = int(mycura.getProperty("machine_extruder_count", "value"))
         machine_name = str(mycura.getProperty("machine_name", "value"))
-                
+        if str(mycura.getProperty("machine_gcode_flavor", "value")) == "RepRap (RepRap)":
+            self._instance.setProperty("jerk_cmd", "value", "reprap_flavor")
+        else:
+            self._instance.setProperty("jerk_cmd", "value", "marlin_flavor")
+
         # Warn the user if the printer is an Ultimaker-------------------------
-        if "Ultimaker" in machine_name:
+        if "Ultimaker" in machine_name or "Griffon" in str(mycura.getProperty("machine_gcode_flavor", "value")):
```

Comment:
```suggestion
        if "Ultimaker" in machine_name or "Griffin" in str(mycura.getProperty("machine_gcode_flavor", "value")):
```

I think we spell it 'Griffin' :-)